tests/integration_tests/test_wave_04_05_06.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+@pytest.mark.integration_test
+def test_integration_wave_04_05_06():
+    camila = Vendor()
+    valentina = Vendor()
+
+    item_clothing1 = Clothing(condition=1.0)
+    item_clothing2 = Clothing(condition=2.0)
+    item_electronics1 = Electronics(condition=1.0)
+    item_electronics2 = Electronics(condition=2.0)
+    item_decor1 = Decor(condition=1.0)
+    item_decor2 = Decor(condition=2.0)
+
+    camila.add(item_electronics1)
+    camila.add(item_clothing1)
+    camila.add(item_clothing2)
+
+    valentina.add(item_electronics2)
+    valentina.add(item_decor1)
+    valentina.add(item_decor2)
+    
+
+    # swap first item
+    result = camila.swap_first_item(valentina)
```

Comment:
I know in test files in practice naming conventions are often a little more lax, but I was wondering if it's worth switching this to something like `swap_result` to more closely describe what it's holding?